packages/admin-next/dashboard/src/v2-routes/regions/region-create/components/create-region-form/create-region-form.tsx

```diff
@@ -78,7 +78,7 @@ export const CreateRegionForm = ({
 
   const { t } = useTranslation()
 
-  const { mutateAsync, isLoading } = useCreateRegion()
+  const { mutateAsync, isPending } = useCreateRegion()
```

Comment:
Why is it `isPending` here? Should it be isPending everywhere or this hook in particular is defined like that?